constants.py

```diff
@@ -476,6 +481,76 @@
 kCameraServoIGain = 0.0
 kCameraServoDGain = 0.0
 
+# shooter parameters
+kTurretMotorId = 22
+kShootingMotorId = 23
+kHoodMotorId = 21
+
+kSimTurretMotorPort = 9
+kSimShootingMotorPort = 15
+kSimHoodMotorPort = 16
+
+kSimTurretMinimumLimitSwitchPort = 0
+kSimTurretMaximumLimitSwitchPort = 1
+kSimHoodMinimumSwitchPort = 2
+kSimHoodMaximumSwitchPort = 3
+
+kShootingPIDSlot = 0
+kShootingPGain = 0.6
+kShootingIGain = 0
+kShootingDGain = 0
+kShootingMotorInverted = True
+
+kShootingMappingFunction = lambda x: 16.41 * math.e ^ (0.3662 * x) + 532
```

Comment:
very kool algorithms. Can these be modified when we get more data points?

Reply:
Most likely will be